### ISSUE TITLE
Release the pre-training weights

### ISSUE THREAD

**Issue by huaibovip** [tzayuan/TransMatch_TMI#7]
Hello, I can not reproduce the results in the paper.

Settings in all datasets,
    NCC: 1,
    Diffusion: 4
    learning rate: 4e-4

IXI DSC: 0.7589

If you release the pre-training weights, I can save a lot of energy.
Thanks.
